services/src/main/java/org/keycloak/services/resources/admin/ClientScopeResource.java

```diff
@@ -148,14 +149,19 @@ public ClientScopeRepresentation getClientScope() {
     @Tag(name = KeycloakOpenAPI.Admin.Tags.CLIENT_SCOPES)
     @Operation(summary = "Delete the client scope")
     public Response deleteClientScope() {
-        auth.clients().requireManage(clientScope);
 
-        try {
-            realm.removeClientScope(clientScope.getId());
-            adminEvent.operation(OperationType.DELETE).resourcePath(session.getContext().getUri()).success();
-            return Response.noContent().build();
-        } catch (ModelException me) {
-            throw ErrorResponse.error(me.getMessage(), Response.Status.BAD_REQUEST);
+        auth.clients().requireManage(clientScope);
+        long ClientScopesCount =  Arrays.stream(realm.getClientScopesStream().toArray()).count();
+        if(ClientScopesCount>1) {
```

Comment:
Minor: formatting. Could you please use spaces? So something like `if (clientScopesCount > 1) {` . This applies for some other places in this PR (Like for example `} else }` etc. Also can be good to rename variable `ClientScopesCount` with lower case at the beginning (java convention to use lowercased variables).